workflow-bot-app/src/main/java/com/symphony/bdk/workflow/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.symphony.bdk.workflow.exception;
+
+import com.symphony.bdk.workflow.api.v1.dto.ErrorResponse;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@Slf4j
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+
+  @ExceptionHandler
+  public ResponseEntity<ErrorResponse> handle(Throwable exception) {
+    log.error("Internal server error: [{}]", exception.getMessage());
+    return handle(exception.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
@yinan-symphony I'm not sure about this http status, can you suggest something better? 
In case the throwable is not an InternalServerError, neither an IllegalArgumentException or a RuntimeException, is will be returned with 500 http status..